### PR TITLE
Add missing Prometheus metrics for validator BFT stats

### PR DESCRIPTION
**Pull request by siy**

### CODE REVIEW

**Review by LukasGasior1 — APPROVED** [review]

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java [hunk 1]
```diff
@@ -243,6 +247,27 @@ private void appendField(StringBuilder builder, String name, Object value) {
 
   private void exportCounters(StringBuilder builder) {
     EXPORT_LIST.forEach(counterType -> generateCounterEntry(counterType, builder));
+
+    getProposalStats()
+        .ifPresent(
+            uptime -> {
+              appendCounter(
+                  builder,
+                  COUNTER_PREFIX + "radix_engine_cur_epoch_completed_proposals",
```
Comment (diegomrsantos) [inline radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java:256]: maybe it is better to extract it to a static constant?

Comment (siy) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java:256]: There is just one location, where this constant is used, so it does not make code more readable or more convenient for support and maintenance.

Comment (diegomrsantos) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java:256]: will this concatenation happen only once or every time this value is updated?

Comment (siy) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java:256]: Compiler is smart enough to optimize it.

**Pull request merged**